### ISSUE TITLE
Myriad reference shown in Red in Rider although it compiles

### ISSUE THREAD

**Issue by 7sharp9** [MoiraeSoftware/myriad#12]
![Screenshot 2019-10-07 at 21 04 35](https://user-images.githubusercontent.com/588746/66344595-2ccb1500-e946-11e9-927d-12cafb98b4f2.png)

Any ideas @enricosada ?
Im sure we had something like this in Falanx?

**Comment by 7sharp9** [comment]
Ionide crashes on load :-(

**Comment by 7sharp9** [comment]
```
/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2542 The F# server crashed 5 times in the last 3 minutes. The server will not be restarted.
onDidNotificationChange @ /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2542
```

**Comment by Krzysztof-Cieslak** [comment]
For Ionide this is fixed by parts of #15 and #16 (set in settings that we want to load `src/Myriad.sln` by default)
